emoji_list.py

```diff
@@ -0,0 +1,2038 @@
+# flake8: noqa
```

Comment:
This came with the file, and is a comment to indicate to [flake8](https://flake8.pycqa.org/en/latest/) not to lint this file.